lib/grape/path.rb

```diff
@@ -22,11 +22,11 @@ def root_prefix
     end
 
     def uses_specific_format?
-      !!(settings[:format] && Array(settings[:content_types]).size == 1)
+      (settings[:format] && Array(settings[:content_types]).size == 1)
```

Comment:
This and the other violation in this file are going to return a `boolean` value anyways, no need for double bang